README.md

```diff
@@ -69,22 +80,26 @@ for (int i = 0; i < toSend.Length; i++)
 Screen.Write(0, 0, 10, 10, toSend);
 ```
 
-The Console class works in a similar way as the classic `System.Console`:
+The Console class works in a similar way as the classic `System.Console` i.e. it writes to the remote Debbuger output on the host (VS):
```

Comment:
No need to bring in Debug.WriteLine to the conversation and the extra explanation... that makes it more confusing...
```suggestion
The Console class works in a similar way as the classic `System.Console`. In order to use it you have to reference it using the fully qualified name of the methods, like this:
```